### ISSUE TITLE
Is this still under maintenance?

### ISSUE THREAD

**Issue by itning** [rapiz1/rathole#318]
It seems that two of them have submitted any new codes.

**Comment by fernvenue** [comment]
Yes, of course. By the way, frequent commits don't necessarily imply maintenance. Many mature projects are in maintenance without frequent commits, because they are already mature enough and require fewer code changes. And...I think this is not an issue, you really need to submit this to discussions not issues.

**Comment by orhun** [comment]
Right now `rathole` cannot be built with Rust 1.80. It would be nice to get a new release with updated deps.

See: https://gitlab.archlinux.org/archlinux/packaging/packages/rathole/-/issues/1

**Comment by deutrino** [comment]
This project looks abandoned.

**Comment by fernvenue** [comment]
This project is still actively maintained, we're currently preparing the next release, and it will be published as planned.

And re to @deutrino directly: infrequent commits don't necessarily mean a lack of maintenance, rathole's feature set is already fairly complete and covers the vast majority of use cases well, so we don't push overly frequent updates or releases just for the sake of activity, that would add churn without real value. This is expected behavior for a mature, stable project rather than a sign of it being abandoned.

We do keep an eye on outstanding issues (including build/toolchain compatibility ones) and will fold relevant fixes into the upcoming release :)

**Comment by fernvenue** [comment]
> Right now `rathole` cannot be built with Rust 1.80. It would be nice to get a new release with updated deps. See: https://gitlab.archlinux.org/archlinux/packaging/packages/rathole/-/issues/1

Hi @orhun, thx for the info, you can open an new issue for this.
